apollo-federation/src/sources/connect/validation/snapshots/validation_tests@invalid-path-parameter.graphql.snap

```diff
@@ -6,7 +6,7 @@ input_file: apollo-federation/src/sources/connect/validation/test_data/invalid-p
 [
     Message {
         code: InvalidUrl,
-        message: "`GET` in `@connect(http:)` on `Query.resources` must be a valid URL template. Variable type must be one of $args, $this, $config, got `$blah`",
+        message: "`GET` in `@connect(http:)` on `Query.resources` must be a valid URL template. Unknown variable namespace `$blah`",
```

Comment:
Can we still list the allowed namespaces as a hint? I guess maybe we should also add a link to variables docs to some errors.

Reply:
This required some significant changes to propagate the invalid namespace error up to a point in the stack that has the expression context, so it can enumerate the available namespaces, but I got it working.